### PR TITLE
Add sort feature to records in webapp UI

### LINKED ISSUE #190: Records in the final UI should be sorted according to the time stamp
**Description:**

Records in the final UI should be sorted according to the timestamp. Refer [1] for the UI

[1] http://ec2-34-238-28-168.compute-1.amazonaws.com:8080/testgrid/v0.9/

### PR DESCRIPTION
**Pull request by azinneera**
## Purpose
Resolve #190 

## Goals

- Add sort feature
  Records are sorted by Start Time in descending order by default. Other fields have also been made 
  sortable.
- Add search and pagination features.

## Security checks

- Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
- Ran FindSecurityBugs plugin and verified report? yes
- Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: web/src/main/webapp/index.html [hunk 1]
```diff
@@ -30,7 +31,7 @@ <h1 class="title">TestGrid Dashboard</h1><hr />
             <table id="product-test-plans" class="table table-striped">
                 <thead>
                 <tr>
-                    <th>ID</th>
+                    <th></th>
```
Comment (vidurananayakkara) [inline web/src/main/webapp/index.html]: Do we need the empty `<th></th>` tag?

Comment (azinneera) [reply web/src/main/webapp/index.html]: Yes, the content is loaded dynamically

File: web/src/main/webapp/js/functions.js [hunk 2]
```diff
@@ -172,17 +191,26 @@ function getTestScenarios (id) {
...
             if (data.length > 0) {
                 var innerHtml = "<tr class='test-scenario hidden'><td colspan=6><h4>Test Scenarios</h4><hr/>"
-                        + "<table class='table test-scenarios'><thead><tr>"
-                        + "<th>ID</th><th>Name</th><th>Status</th></tr></thead><tbody>";
+                        + "<table id='test-scenarios' class='table test-scenarios'><thead><tr>"
+                        + "<th></th><th>Name</th><th>Status</th></tr></thead><tbody>";
                 for (var i = 0; i < data.length; i++) {
-                    innerHtml += "<tr id = " + data[i].id + "><td><button class='expand-test-cases'>+</button>&nbsp;"
-                        + data[i].id + "</td><td>"
+                    innerHtml += "<tr id = " + data[i].id + "><td><button class='expand-test-cases'>+</button></td><td>"
```
Comment (sameerawickramasekara) [inline web/src/main/webapp/js/functions.js]: Since we already use Jquery, cant we use Jquery append here

Comment (azinneera) [reply web/src/main/webapp/js/functions.js]: For jquery append, the insertion is done as the last element of the specified parent. Therefore, [jquery after](http://api.jquery.com/after/) is more appropriate for this. For clarity, the string catenation is done outside the jquery function.

Comment (sameerawickramasekara) [reply web/src/main/webapp/js/functions.js]: (Y)

**Pull request merged**